pnc-model/src/main/java/org/jboss/pnc/model/Environment.java

```diff
@@ -1,14 +1,15 @@
 package org.jboss.pnc.model;
 
-import javax.persistence.*;
+import org.hibernate.envers.Audited;
 
-import java.io.Serializable;
+import javax.persistence.*;
 
 /**
  * The Class Environment.
  */
 @Entity
-public class Environment implements Serializable {
+@Audited
```

Comment:
I assumed that for reproducing build we need a configuration plus an 
environment. That's why I added @Audited to the Environment.

But I totally agree - we may consider doing it immutable.

On 04/08/2015 04:22 PM, Paul Gier wrote:
>
> In pnc-model/src/main/java/org/jboss/pnc/model/Environment.java 
> <https://github.com/project-ncl/pnc/pull/278#discussion_r27973788>:
>
> >
> >  /**
> >   * The Class Environment.
> >   */
> >  @Entity
> > -public class Environment implements Serializable {
> > +@Audited
>
> I'm wondering if environment should be immutable instead of auditing it.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/project-ncl/pnc/pull/278/files#r27973788>.
>